website/src/roadmap/sql.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: section
+title: "Beam SQL Roadmap"
+permalink: /roadmap/sql/
+section_menu: section-menu/roadmap.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# SQL
```

Comment:
@akedin @amaliujia @apilloud @XuMingmin @xumingming @vectorijk I did not write much here at the moment. FYI there is a place here for advertising exciting SQL efforts to Beam users.

http://apache-beam-website-pull-requests.storage.googleapis.com/6718/roadmap/sql/index.html

Reply:
It's also great to have the status of Beam SQL running on different runners.